pkg/kubectl/cmd/patch.go

```diff
@@ -331,5 +331,5 @@ func patchOperation(didPatch bool) string {
 	if didPatch {
 		return "patched"
 	}
-	return "not patched"
+	return "not patched (no change)"
```

Comment:
I still think "not patched" is still misleading when a patch was actually submitted to the server... "patched (no change)" seems more accurate to me

Reply:
> I still think "not patched" is misleading when a patch was actually submitted to the server... "patched (no change)" seems more accurate to me

agreed. done.